### DISCUSSION TITLE
Ideas

### DISCUSSION THREAD

**Discussion by CobaltonGit** [maxnut/GDmenu#229] [Ideas]
This idea will be true **ONLY FOR 2.2!!!**

Platformer-related hacks (e.g. Force Platformer) are already present in Prism Menu (from Geode develpoers) and Megahack v8, but why not add them to your mod menu as well? It was popular when 2.1 was still around and was even better than GDHM, so it's best to say that platformer-related hacks in your menu will make GDMO popular (if you get lucky of course).

**Comment by vo0ov** [comment]
Please add a function to change the game mode like in Mega Hack.
